views/frontend/global-embed.js.php

```diff
@@ -0,0 +1,18 @@
+<?php // Check that the WP Admin user specified a Site ID, otherwise log a console error.
+if (empty($data['site_id'])) : ?>
+<script type="text/javascript">console.error(
+  'No Sitka Insights Site ID detected! Please specify one at <?= WP_SITEURL ?>/wp-admin/options-general.php?page=sitka-insights'
+);</script>
+<?php else : ?>
+<!-- Sitka Insights global embed -->
+<script type="text/javascript">
+  (function (g, e, a, r, l, b) {
```

Comment:
Is the embed snippet example in [SCS-970](https://sitecrafting.atlassian.net/browse/SCS-970) outdated? Looks like the snippet you're using here is slightly different (i.e. passing in the site id as a parameter instead of hardcoding it in the script).

This isn't an issue. I'm mostly just curious if we need to update documentation somewhere to use this version of the embed snippet, since it seems like a better way of handling the site id than the snippets in the issue.

Reply:
No, as far as I know the snippet from the Jira ticket is up-to-date. I took the initiative to make this change because I didn't want to add another level of abstraction (i.e. rendering the snippet in a template - whereas I assume on most current installs we're hard-coding it) and put a template-rendered value in the middle of some partially obfuscated code, which seemed like a proper foot-gun.

IMHO for manual installations (without this plugin) the existing snippet is fine.